src/auth/auth.module.ts

```diff
@@ -9,7 +9,7 @@ import { AtStrategy } from "./strategies/at.strategy";
 import { RtStrategy } from "./strategies/rt.strategy";
 import { DiscordStrategy } from "./strategies/discord.strategy";
 import { DiscordAuthService } from "./discord-auth.service";
-import { EmailService } from "src/utils/emails/sendEmail";
+import { EmailService } from "@/utils/emails/email.service";
```

Comment:
these `@` are causing issue in e2e tests, they can only read relative paths🥲 
We can possibly fix it, `@/` looks a lot nicer, but it can be for another PR
https://stackoverflow.com/questions/64028100/nestjs-jest-cannot-find-module-with-absolute-path